data.json

```diff
@@ -125,6 +125,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "Flutterwave",
+    "difficulty": "hard",
+    "description": "Contribute 2 or more merged PRs to any of our community libraries and win a shirt",
+    "reference": "https://twitter.com/Ace_KYD/status/1183738026511519745",
+    "image": "https://pbs.twimg.com/media/EG17czRWsAA9PKF?format=jpg&name=4096x4096",
```

Comment:
Please rehost this image, you can use imgur for example.

Reply:
Updated =]